Events/services.js

```diff
@@ -0,0 +1,28 @@
+var response = require('../Objects/response.js');
+
+var createEvent = function (req){
+  //return json containing eventuid
```

Comment:
If this comment is intended to document this function, it should probably go on the line above the function declaration.  We'll be talking about using JSDoc for this sort of thing this Saturday.